client/src/components/overview/parts/imageGallery/ImageGallery.jsx

```diff
@@ -9,137 +9,65 @@ class ImageGallery extends React.Component {
   constructor(props) {
     super(props)
 
-    this.state = {
-      expanded: false,
-      styles: [],
-      current: [],
-      mainIndex: 0,
-      amount: 0,
-      currentThumbnails: []
-    }
-    this.getImages = this.getImages.bind(this);
-    this.updateMainPic = this.updateMainPic.bind(this);
-    this.mainSlide = this.mainSlide.bind(this);
+    this.handleClick = this.handleClick.bind(this)
   }
 
-  mainSlide(dir) {
-
-    if (this.state.mainIndex === 0 && dir === -1) {
-      return
-    }
-
-    if (this.state.mainIndex === this.state.amount - 1 && dir > 0) {
-      return
-    }
+  handleClick(dir) {
```

Comment:
I suggest a more meaningful name, this is probably set to an onClick event, so in the render we already know it's a click handler.

May I suggest "nextImage"